networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/GossipForkManager.java

```diff
@@ -188,6 +190,15 @@ public synchronized void publishDataColumnSidecar(final DataColumnSidecar dataCo
         GossipForkSubscriptions::publishDataColumnSidecar);
   }
 
+  public void publishExecutionProof(final ExecutionProof executionProof) {
+    // for now we don't have a slot in the message data (execution proof) to use
+    // I believe it's safe to just check the current epoch
+    LOG.debug("Publishing execution proof for slot {}", executionProof);
```

Comment:
We don't need this log here. We are already logging it on `ExecutionProofGossipManager`.